addons/mass_mailing_sms/models/mailing_list.py

```diff
@@ -27,6 +27,20 @@ def action_view_contacts_sms(self):
         action['context'] = dict(action.get('context', {}), search_default_filter_valid_sms_recipient=1)
         return action
 
+    def action_send_new_sms(self):
+        return {
+            "type": "ir.actions.act_window",
+            "name": _("Send Sms From Mailing List"),
+            "res_model": "mailing.mailing",
+            "view_mode": "form",
+            "views": [(self.env.ref("mass_mailing_sms.mailing_mailing_view_form_sms").id, "form")],
+            "context": {
+                'default_contact_list_ids': [self.id],
+                'default_mailing_type': 'sms',
+                'mailing_sms': True,
```

Comment:
What is this "mailing_sms" key ?

Reply:
This key is passed already while creating SMS mailing (from landing action), see https://bit.ly/3KavCxi and is used at several places for SMS mailings. So did not want to loose anything coming from our new action 🙂